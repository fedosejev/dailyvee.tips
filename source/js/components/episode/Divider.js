import React from 'react';

const Divider = () => <span className="divider"> / </span>;

export default Divider;
