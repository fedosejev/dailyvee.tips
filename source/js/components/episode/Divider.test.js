import React from 'react';
import Divider from './Divider';
import renderer from 'react-test-renderer';

describe('Divider', () => {
  test('SHOULD render correctly', () => {
    const tree = renderer.create(
      <Divider />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
