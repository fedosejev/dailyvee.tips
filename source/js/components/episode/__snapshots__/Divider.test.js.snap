// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider SHOULD render correctly 1`] = `
<span
  className="divider"
>
   / 
</span>
`;
